### PR TITLE
Update ReadMe Content

### PR DESCRIPTION
**Pull request by HaedrichHC**
Here the right ReadMe Update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +38,32 @@ We are using [Turborepo](https://turborepo.org/) to compile and build our applic
...
+
+### Contributing
+
+Planning in process
+
+---
+
+### License
 
-#### Frontend
```
Comment (hkd987) [inline README.md:31]: Lets keep frontend in here as a placeholder,

File: README.md [hunk 2]
```diff
@@ -22,10 +38,32 @@ We are using [Turborepo](https://turborepo.org/) to compile and build our applic
...
+>#### New Backend Project
 
 Using the `bootstrap` mutation you can setup a brand new workspace for you and your project's users.
 
-#### IFrame
+>#### IFrame
+
+Planning in process
+
+>#### Frontend
```
Comment (hkd987) [inline README.md:49]: Im not sure we want to keep the block quotes here.

File: README.md [hunk 3]
```diff
@@ -22,10 +38,32 @@ We are using [Turborepo](https://turborepo.org/) to compile and build our applic
 - 5432: Has a docker container with Postgres running
 - 4000: Will have the node app, after starting it with `npm run start:dev` using default settings
 
-##### New Backend Project
+>#### New Backend Project
 
 Using the `bootstrap` mutation you can setup a brand new workspace for you and your project's users.
 
-#### IFrame
+>#### IFrame
```
Comment (hkd987) [inline README.md:45]: Im not sure we want to keep the block quotes here.

File: README.md [hunk 4]
```diff
@@ -22,10 +38,32 @@ We are using [Turborepo](https://turborepo.org/) to compile and build our applic
 - 5432: Has a docker container with Postgres running
 - 4000: Will have the node app, after starting it with `npm run start:dev` using default settings
 
-##### New Backend Project
+>#### New Backend Project
```
Comment (hkd987) [inline README.md:41]: Im not sure we want to keep the block quotes here.

File: README.md [hunk 5]
```diff
@@ -1,19 +1,35 @@
...
 ### Developer Setup (WIP)
 
-#### Notes:
+>#### Notes
 
 - As this project is currently not feature stable, no postgres tables are currently made when running the backend application
 - Each section of the application has its own `package.json` and `tsconfig.json`
 
-#### Backend
+>#### Backend
```
Comment (hkd987) [inline README.md:32]: Im not sure we want to keep the block quotes here.

File: README.md [hunk 6]
```diff
@@ -1,19 +1,35 @@
...
 ### Deployment (WIP)
 
 We are using [Turborepo](https://turborepo.org/) to compile and build our application.
 
+---
+
 ### Developer Setup (WIP)
 
-#### Notes:
+>#### Notes
```
Comment (hkd987) [inline README.md:27]: Im not sure we want to keep the block quotes here.

File: README.md (hunk 1 above)
Comment (HaedrichHC) [reply README.md:31]: ok

File: README.md (hunk 2 above)
Comment (HaedrichHC) [reply README.md:49]: was such an idea but if you do not like it, I put it out again. Should I delete `Frontend` here then?

File: README.md (hunk 3 above)
Comment (HaedrichHC) [reply README.md:45]: was such an idea but if you do not like it, I put it out.

File: README.md (hunk 4 above)
Comment (HaedrichHC) [reply README.md:41]: was such an idea but if you do not like it, I put it out.

File: README.md (hunk 5 above)
Comment (HaedrichHC) [reply README.md:32]: was such an idea but if you do not like it, I put it out again

File: README.md (hunk 6 above)
Comment (HaedrichHC) [reply README.md:27]: was such an idea but if you do not like it, I put it out.

File: README.md (hunk 2 above)
Comment (hkd987) [reply README.md:49]: We just need to make sure that we have `Frontend` somewhere in the readme where it makes the most sense.

**Review by hkd987 — APPROVED** [review]

**Comment by hkd987** [comment]
Related to https://github.com/authsio/core/issues/9

**Pull request merged**